forms/gridfield/GridFieldAddExistingAutocompleter.php

```diff
@@ -90,7 +90,7 @@ public function __construct($targetFragment = 'before', $searchFields = null) {
 	/**
 	 *
 	 * @param GridField $gridField
-	 * @return string - HTML
+	 * @return string[] - HTML
```

Comment:
The compressed diff is misleading you into thinking that the `Convert::array2json` is the return of the method this doc block is for (it's not). [See line 124](https://github.com/silverstripe/silverstripe-framework/pull/5901/files#diff-c2c534971b8d2554baeba1399c70d88aR124).